control_panel_api/tests/test_permissions.py

```diff
@@ -7,25 +7,21 @@
     HTTP_201_CREATED,
     HTTP_204_NO_CONTENT,
     HTTP_403_FORBIDDEN,
-    HTTP_404_NOT_FOUND,
 )
 from rest_framework.test import APITestCase
 
-from control_panel_api.aws import aws
 from control_panel_api.models import AppS3Bucket
 
 
-@patch.object(aws, 'client', MagicMock())
+@patch('control_panel_api.aws.aws.client', MagicMock())
 class AppPermissionsTest(APITestCase):
```

Comment:
I wonder if it's a good time to move these last 4 test case classes in separate files under tests/permissions as well.